core/src/main/java/com/graphhopper/routing/ch/EdgeBasedWitnessPathSearcher.java

```diff
@@ -87,12 +85,11 @@
     // data structures used to build the shortest path tree
     // we allocate memory for all possible edge keys and keep track which ones have been discovered so far
     private double[] weights;
-    private int[] edges;
-    private int[] incEdges;
+    private int[] prepareEdges;
```

Comment:
Here we got rid of one of the int[] arrays, i.et 4byte*N where N=2*number of base graph edges. We no longer need to store the incEdges, because we use the edge keys directly now and the index position equals the edge key for each element.